serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -75,14 +77,9 @@ RUN pip install torch==${TORCH_VERSION} transformers==${transformers_version} ac
     pip3 cache purge
 
 # Install TensorRT and TRT-LLM Deps
-RUN pip install --no-cache-dir --extra-index-url https://pypi.nvidia.com tensorrt==${tensorrtlibs_version} && \
+RUN pip install --no-cache-dir --extra-index-url https://pypi.nvidia.com tensorrt==${tensorrtlibs_version} nvidia-ammo~=${ammo_version} janus==${janus_version} optimum==${optimum_version} diffusers==${diffusers_version} && \
```

Comment:
need to remove optimum and diffusers dependencies

Reply:
Fixed, thanks